carmin-server/server/test/resources/test_execution.py

```diff
@@ -39,3 +43,94 @@ def test_get_invalid_execution(self, test_config):
             })
         error = error_from_response(response)
         assert error == EXECUTION_NOT_FOUND
+
+    def test_put_valid_execution_name_and_timeout(self, test_config,
+                                                  execution_id):
+        response = test_config.test_client.put(
+            '/executions/{}'.format(execution_id),
+            headers={"apiKey": standard_user().api_key},
+            data=json.dumps(PATCH_VALID_EXECUTION))
+        assert response.status_code == 204
+
+        response = test_config.test_client.get(
+            '/executions/{}'.format(execution_id),
+            headers={
+                "apiKey": standard_user().api_key
+            })
+        execution = load_json_data(response)
+        assert execution["name"] == PATCH_VALID_EXECUTION["name"]
+        assert execution["timeout"] == PATCH_VALID_EXECUTION["timeout"]
+
+    def test_put_valid_execution_name(self, test_config, execution_id):
+        response = test_config.test_client.put(
+            '/executions/{}'.format(execution_id),
+            headers={"apiKey": standard_user().api_key},
+            data=json.dumps(PATCH_VALID_EXECUTION2))
+        assert response.status_code == 204
+
+        response = test_config.test_client.get(
+            '/executions/{}'.format(execution_id),
+            headers={
+                "apiKey": standard_user().api_key
+            })
+        execution = load_json_data(response)
+        assert execution["name"] == PATCH_VALID_EXECUTION2["name"]
+
+    def test_put_valid_execution_timeout(self, test_config, execution_id):
+        response = test_config.test_client.put(
+            '/executions/{}'.format(execution_id),
+            headers={"apiKey": standard_user().api_key},
+            data=json.dumps(PATCH_VALID_EXECUTION3))
+        assert response.status_code == 204
+
+        response = test_config.test_client.get(
+            '/executions/{}'.format(execution_id),
+            headers={
+                "apiKey": standard_user().api_key
+            })
+        execution = load_json_data(response)
+        assert execution["timeout"] == PATCH_VALID_EXECUTION3["timeout"]
```

Comment:
Please make sure the change took place in the DB. You can find an example in test_register_already_existing_user_folder (register_test.py). This will insure that the DB changes really took place.